front/app/containers/CustomPageShow/CustomPageHeader.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import FullWidthBannerLayout from './FullWidthBannerLayout';
+
+const CustomPageHeader = ({
+  headerLayout,
+  header_bg,
+  headerMultiloc,
+  subheaderMultiloc,
+  headerColor,
+  headerOpacity,
+}) => {
+  return (
+    <>
+      {headerLayout === 'full_width_banner_layout' && (
+        <FullWidthBannerLayout
+          imageUrl={header_bg.large}
+          imageColor={headerColor}
+          imageOpacity={headerOpacity}
+          headerMultiloc={headerMultiloc}
+          subheaderMultiloc={subheaderMultiloc}
```

Comment:
So here too, no need to pass all these as props if you just use the slug from the router and load it inside the components that need the values.